build/package/NOTICE

```diff
@@ -44,13 +44,18 @@ The following NOTICE information applies to binary components distributed with t
   This product includes software from the Spring Framework,
   under the Apache License 2.0 (see: StringUtils.containsWhitespace())
 
+- org.apache.daffodil.daffodil-cli_<VERSION>.jar
 - org.apache.daffodil.daffodil-core_<VERSION>.jar
 - org.apache.daffodil.daffodil-io_<VERSION>.jar
+- org.apache.daffodil.daffodil-japi_<VERSION>.jar
 - org.apache.daffodil.daffodil-lib_<VERSION>.jar
 - org.apache.daffodil.daffodil-runtime1-layers_<VERSION>.jar
 - org.apache.daffodil.daffodil-runtime1-unparser_<VERSION>.jar
 - org.apache.daffodil.daffodil-runtime1_<VERSION>.jar
-- org.apache.daffodil.daffodil-sapi_<VERSION>.jar
+- org.apache.daffodil.daffodil-runtime2_<VERSION>.jar
+- org.apache.daffodil.daffodil-sapi_<VERSION>.ja
+- org.apache.daffodil.daffodil-tdml-lib_<VERSION>.jar
+- org.apache.daffodil.daffodil-tdml-processor_<VERSION>.jarr

```

Comment:
I think these changes are incorrect. This is the list of daffodil jars that end up in the daffodil-debugger.zip:
```
org.apache.daffodil.daffodil-core_2.12-3.4.0.jar
org.apache.daffodil.daffodil-io_2.12-3.4.0.jar
org.apache.daffodil.daffodil-lib_2.12-3.4.0.jar
org.apache.daffodil.daffodil-runtime1-layers_2.12-3.4.0.jar
org.apache.daffodil.daffodil-runtime1-unparser_2.12-3.4.0.jar
org.apache.daffodil.daffodil-runtime1_2.12-3.4.0.jar
org.apache.daffodil.daffodil-sapi_2.12-3.4.0.jar
org.apache.daffodil.daffodil-udf_2.12-3.4.0.jar
```

Only those should be mentioned in this NOTICE file, which is what you had before and was correct. I think I just misunderstood what you were saying in a previous comment.

You only need to include LICENSE/NOTICE information if the jar ends up in the daffodil-debugger zip. The daffodil-cli binary is just a useful place to look for that information because it's a superset of the jars that daffodil-debugger uses and we've already done the work to dig up license/notice information.

And it looks like all the updates made to the Daffodil CLI license/files were for transitive dependencies of the CLI that daffodil-debugger doesn't have. So I think no changes are actually needed.

Your NONOTICE changes look fine though.

Sorry for the miscommunication, I guess I wasn't very clear what I was asking about.